packages/modules/src/event/EventModel.ts

```diff
@@ -129,16 +129,16 @@ EventSchema.methods = {
     }
   },
 
-  eventIsPublished() {
-    return this.dateEnd > new Date();
+  eventIsPublished(dateEnd) {
+    return dateEnd > new Date();
   },
 
   capitilizeName(str: string): string {
     return str.charAt(0).toUpperCase() + str.slice(1);
   },
 
-  getCurrentPrice() {
-    return this.prices.find((price) => new Date() < price.date);
+  getCurrentPrice(prices) {
+    return prices.find((price) => new Date() < price.date);
   },
 };
```

Comment:
why is this not a gql resolver?

Reply:
I use this logic in another function then I made a model method